doc/DeveloperGuide.md

```diff
@@ -56,11 +56,14 @@ Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (un
 Priority | As a ... | I want to ... | So that I can...
 -------- | :-------- | :--------- | :-----------
 `* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
-`* * *` | user | add a new person |
+`* * *` | new user | see examples of each command | replicate the command but with information relevant to my contacts
+`* * *` | user | add a new person | increase the amount of people in my address book
```

Comment:
"Increasing the amount ..." that's a programming perspective. What's the user story here, why does he/she want to do this?